tools/validators/instance_validator/validate/handler.py

```diff
@@ -108,33 +127,58 @@ def RunValidation(filenames: List[str],
   """Master runner for all validations."""
   saved_stdout = sys.stdout
   report_file = None
+
+  print('[INFO]\t{time}\tStarting validation process.'
+        .format(time=datetime.datetime.now())
+        )
   if report_filename:
     # pylint: disable=consider-using-with
     report_file = open(report_filename, 'w', encoding='utf-8')
     sys.stdout = report_file
   try:
-    print('\nStarting validator...\n')
-    print('\nStarting universe generation...\n')
+    print('[INFO]\t{time}\tGenerating ontology.'
+          .format(time=datetime.datetime.now())
+          )
     universe = generate_universe.BuildUniverse(
         use_simplified_universe=use_simplified_universe,
         modified_types_filepath=modified_types_filepath)
     if not universe:
-      print('\nError generating universe')
+      print('[ERROR]\t\tUniverse did not generate properly.')
       sys.exit(0)
-    print('\nStarting config validation...\n')
+    print('[INFO]\t{time}\tOntology generated.'
+          .format(time=datetime.datetime.now())
+          )
     entities = _ValidateConfig(filenames, universe, is_udmi)
     if subscription:
-      print('\nStarting telemetry validation...\n')
+      print('[INFO]\t{time}\tStarting telemetry validation.'
+            .format(time=datetime.datetime.now())
+            )
       _ValidateTelemetry(subscription, service_account, entities,
                          timeout, is_udmi)
     else:
-      print('Subscription is needed for telemetry validation')
-
+      print('[WARNING]\t{time}\tTelemetry validation skipped, subscription '
+            'not found. Please provide a subscription and service account to '
+            'run telemetry validation. See here for more details: '
+            'https://github.com/google/digitalbuildings/tree/master/tools/'
+            'validators/instance_validator#telemetry-validation'
```

Comment:
nit use the github io link: https://google.github.io/digitalbuildings/tools/validators/instance_validator/#telemetry-validation

Reply:
fixed